scripts/prepare_sector_network.py

```diff
@@ -1856,6 +2014,9 @@ def remove_h2_network(n):
     if "noH2network" in opts:
         remove_h2_network(n)
 
+    if not options["biomass_transport"]:
+        remove_biomass_transport(n)
+
```

Comment:
Hmm, it's a bit odd to add biomass transport first and then remove it later if `biomass_transport: False` (same for H2 network above actually). Could we do something like:

```suggestion
    if options["biomass_transport"]:
        add_biomass_transport(n)
    else:
        add_biomass_copperplated(n)
```
where `add_biomass_copperplated(n)` is the current `add_biomass(n)` and `add_biomass_transport(n) includes the changes you made in this PR?

Reply:
yes, that would be more logical. Should we deal with it in a seperate pull-request and change the way the H2 network is implemented as well? I would suggest, that the functions (e.g. add_biomass_copperplate) only add the biomass node(s). We could then all the other functions e.g. add_industry(n) by filtering for all buses with the corresponding carrier ("solid biomass" or "H2"), e.g.
`biomass_nodes = n.buses[n.buses.carrier=="solid biomass]`
 and using them for adding additional components to the network.

Reply:
Yes, let's move that to a separate issue. I just updated everything to the revision #139

Reply:
See proposal in #152